pyre/pyre-interpreter/include/pyre3.14t/modsupport.h

```
@@ -0,0 +1,611 @@
+extern "C" {
+#endif
+/* ── The variadic entry points ─────────────────────────────────────────
+ *
+ * `PyArg_ParseTuple`, `Py_BuildValue` and `PyErr_Format` are C functions with
+ * C bodies, as they are upstream (`pypy/module/cpyext/src/getargs.c`).  Pyre
+ * ships no companion library, so they are `static inline` here and every
+ * extension compiles its own copy; each one is built entirely out of the
+ * non-variadic entry points declared above.
+ */
+
+static inline void _PyPyre_ArgError(const char *fname, const char *message)
```

Clang (14.0.6)

[warning] declaration uses identifier '_PyPyre_ArgError', which is a reserved identifier

(bugprone-reserved-identifier)